skrifa/src/outline/autohint/latin/blues.rs

```diff
@@ -527,4 +673,29 @@ mod tests {
         ];
         assert_eq!(values, &expected);
     }
+
+    #[test]
+    fn cjk_blues() {
+        let font = FontRef::new(font_test_data::NOTOSERIFTC_AUTOHINT_METRICS).unwrap();
+        let style = &style::STYLE_CLASSES[super::StyleClass::HANI];
+        let blues = super::compute_cjk_blues(&font, &[], style);
+        let values = blues[1].as_slice();
+        let expected = [
+            UnscaledBlue {
+                position: 837,
+                overshoot: 824,
+                ascender: 0,
+                descender: 0,
+                flags: blue_flags::TOP,
+            },
+            UnscaledBlue {
+                position: -78,
+                overshoot: -66,
+                ascender: 0,
+                descender: 0,
+                flags: 0,
+            },
+        ];
+        assert_eq!(values, &expected);
+    }
```

Comment:
Seems a little light, I suppose we plan to lean on fuzzing and fauntlet for more substantial testing?

Reply:
Yes, the plan is to use broad scale testing to detect bugs which may be used to inform additional unit tests.